content/en/docs/reference/configuration/scim-response/_index.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "Handling SCIM response"
+linkTitle: "SCIM response"
+weight: 6
+---
+
+IAM already supports a set of [SCIM APIs](../../api/scim-api/_index.md) based on the System for Cross-domain
+Identity Management standard. Starting from IAM v1.9.0, the API response to GET requests on user details
+(i.e. `/scim/Me` and `/scim/Users`) are extended with more information.
+All the new details are included into the _IndigoUser_ field.
+To maintain backward compatibility, in order to enable this feature,
```

Comment:
To enable this feature, dedicated properties (or environment variables) must be used to maintain backward compatibility.